README.md

```diff
@@ -41,7 +41,7 @@ The endpoint can be configured in the Yoti Dashboard when you create/update your
 The image below shows how your application back-end and Yoti integrate into the context of a Login flow.
 Yoti SDK carries out for you steps 6, 7 and the profile decryption in step 8.
 
-![alt text](https://github.com/getyoti/node/raw/master/login_flow.png "Login flow")
+![alt text](https://github.com/getyoti/yoti-node-sdk/raw/master/login_flow.png "Login flow")
```

Comment:
I feel that this shouldn't link to the Node SDK. Can you copy the image across and link it locally? Although I think at some point we will remove the technical bits from the plugin'a readme and just link to the PHP SDK.

Reply:
Good point, I've updated it now with the last check in